projects/amphor/index.js

```diff
@@ -20,12 +25,20 @@ const claimableSilo = '0x06eCFaAde8fcb8C1bC58CB05104604282f8a8144';
 const pendingSilo = '0x361a027e660844f336d5fa07E4cb38c40d5880d9';
 const WETH = '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2';
 
+const totalAssetsVaults = config.ethereum.lvTokens.totalAssets;
+const totalSupplyVaults = config.ethereum.lvTokens.totalSupply;
+
 const tvl = sdk.util.sumChainTvls([
   sumERC4626VaultsExport({
-    vaults: Object.values(config.ethereum.lvTokens),
+    vaults: totalAssetsVaults,
     tokenAbi: 'asset',
     balanceAbi: 'totalAssets',
   }),
+  sumERC4626VaultsExport({
+    vaults: totalSupplyVaults,
+    tokenAbi: 'asset',
+    balanceAbi: 'totalSupply',
```

Comment:
hi @MaksimKrukovichWeb3 

thanks for the PR, but why is this totalSupply and not totalAssets like the rest?

Reply:
hi @g1nt0ki 
the function was changed because the supply is synchronous now